action.yml

```diff
@@ -17,78 +17,75 @@ inputs:
     description: "API account token to retrieve your remote Codacy configuration for the project being analyzed"
   codacy-api-base-url:
     required: false
-    description: "Codacy's API URL, to retrieve your remote Codacy configuration"
+    description: "Codacy API URL to retrieve your remote Codacy configuration"
   format:
     required: false
-    description: "Format of the output file"
+    description: "Output file format"
   output:
     required: false
-    description: "Send analysis results to an output file"
+    description: "Path to a file to save the analysis results"
```

Comment:
I tested this and I suspect that this only works for a local CLI analysis with file system access. It seems to [timeout on a GitHub workflow](https://github.com/codacy-docs/sandbox/actions/runs/3639320751).
Should this even be flagged as supported? Are there other options like this one?

Reply:
Perhaps with the client side tools integration? Haven't tested that one...

Reply:
I'm not sure about this either. :thinking: Let's keep this thread open and we'll check again with some more time later.

Reply:
It indeed worked this time; it was another issue altogether.
I'm still not seeing the file, although there are mentions in the logs. I assume that it's saved on some temporary container storage and then discarded.

Should we mention this?

Reply:
Yes, the file is probably created correctly at the end of the Codacy Analysis CLI run. But I don't think it's necessary to further clarify this, as it isn't even very clear why anyone would want to use this option (save for maybe uploading it as an artifact of the workflow that could then be consulted later for debugging purposes.).